src/components/Recover.tsx

```
@@ -131,6 +136,28 @@ const Recover: React.FC<RecoverProps> = ({
           setIsGroupValid(validation.isValid);
           setGroupError(validation.message);
           
+          // Decode group to set currentThreshold and currentTotalShares
+          if (validation.isValid) {
+            try {
+              const decodedGroup = decode_group(matchingGroup);
+              if (decodedGroup && typeof decodedGroup.threshold === 'number') {
+                setCurrentThreshold(decodedGroup.threshold);
+              }
+              if (decodedGroup && decodedGroup.commits && Array.isArray(decodedGroup.commits)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)